stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/utils/extensions/Filters.kt

```diff
@@ -0,0 +1,16 @@
+package com.getstream.sdk.chat.utils.extensions
+
+import io.getstream.chat.android.client.api.models.FilterObject
+import io.getstream.chat.android.client.models.Filters
+import io.getstream.chat.android.client.models.User
+import io.getstream.chat.android.core.internal.InternalStreamChatApi
+
+@InternalStreamChatApi
+public fun Filters.defaultChannelListFilter(user: User?): FilterObject {
+    val userFiler = if (user == null) neutral() else `in`("members", listOf(user.id))
```

Comment:
🤔 Should we use a `neutral` filter if a user is not set? Consider the following scenario:
1. I'm connecting the user at the same time that I'm showing ChannelListView
2. Channels are loaded faster than the user is connected - I can see channels in which I'm not the member, but I can't view them because I don't have permissions
3. Channels are reloaded because the user was connected - wrong channels disappear